### PR TITLE
Ubuntu build

### PR DESCRIPTION
**Pull request by smoralesd**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000 **N/A**
- [ ] Added relevant unit test for your changes. (`npm run test`) **N/A**
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage` **N/A**
- [ ] Ran precheckin (`npm run precheckin`) **N/A**
- [ ] Added screenshots/GIFs for UI changes. **N/A**

#### Description of changes

Add new job section to build.yaml in order to build our extension on ubuntu agent.

#### Notes for reviewers

General idea is to make sure our extension build on linux. Running this job should increase our confidence that our extension actually run on linux/chrome.
First build run this job in 1m 43s. Also notice it runs in parallel with the other existing jobs.

##### Open question
I'm basically running:
- `npm ci`
- `npm run build:all`

No tests, not tslint, no format check and no header checks since this steps are currently being run on the other jobs. **Is this ok, or do you think we should run addition steps on this job?**

### CODE REVIEW

**Comment by peterdur** [comment]
I think it would be good to run the e2e tests on Ubuntu, if only to check that the extension loads successfully, given that it was broken for a while.

File: build.yaml [hunk 1]
```diff
@@ -117,3 +117,26 @@ jobs:
                 pathtoPublish: '$(System.DefaultWorkingDirectory)/drop'
                 artifactName: 'drop'
             displayName: publish drop
+
+    - job: 'ubuntu_build'
+
+      pool:
```
Comment (devaradhanm) [inline build.yaml]: did u check if there is a way to import yaml file to avoid duplicate code?

Comment (smoralesd) [reply build.yaml]: Nop. It's a good point, a little out of the discussion scope but I'll have a look

Comment (smoralesd) [reply build.yaml]: Done... I think we can take a second look a this later as right now my objective is to make sure we add a new job to run things under ubuntu!

**Comment by smoralesd** [comment]
> I think it would be good to run the e2e tests on Ubuntu, if only to check that the extension loads successfully, given that it was broken for a while.

Agree. Based on @devaradhanm comment, I used template feature to extract the build and run e2e test to a separated file and reuse the template on both, macOS & ubuntu runs. Only the macOS run publish test results and the drop for now.

**Pull request closed without merging**